lib/index.js

```diff
@@ -226,6 +226,14 @@ FluxApp.prototype.rehydrate = function rehydrate(state) {
   return this;
 };
 
+FluxApp.prototype.setPlatform = function setPlatform(platform) {
+  if (platform === 'browser') {
+    require('./platforms/browser')(this);
+  } else if (platform === 'node') {
+    require('./platforms/node')(this);
```

Comment:
what about providing a secondary optional argument for options, like brower#containerId etc?